uaclient/status.py

```diff
@@ -648,4 +650,18 @@ def format_json_status(status: "Dict[str, Any]") -> str:
         or name == "UA_CONFIG_FILE"
     ]
 
-    return json.dumps(status)
+    contract_name = status.pop("subscription", "")
+    contract_id = status.pop("subscription-id", "")
+    created_at = status.pop("contract-created-at", "")
+    tech_lvl_support = status.pop("techSupportLevel", "")
+    products = status.pop("products", [])
+
+    status["contract"] = {
+        "id": contract_id,
+        "name": contract_name,
+        "created_at": created_at,
+        "tech_level_support": tech_lvl_support,
```

Comment:
I think we want to leave the word order the same as it was before, just snake case now, so `tech_support_level`